src/sentry/dynamic_sampling/models/utils.py

```diff
@@ -12,47 +11,177 @@ class DSElement:
     new_sample_rate: float = 0.0
 
 
-def total_elements(ds_elements: List[DSElement]) -> float:
-    ret_val = 0.0
-    for element in ds_elements:
-        ret_val += element.count
-    return ret_val
+def adjust_sample_rates(
+    classes: List[DSElement],
```

Comment:
nit: I don't think `DSElement` is a good name, maybe something like `AdjustableClass`. What do you think?

Reply:
I don't think either are good names `DSElement` has the small advantage that it is shorter it is slightly more
to the point.

AdjustableClass` really doesn't tell you anything except that it is a class and it is in some way adjustable 🙂. 
I particularly dislike having class names containing the word "class" it's just doesn't bring any useful info. 
	
Your point is valid though,It's worth thinking of a better name.

Reply:
I really struggled even to come up with a shitty name like `AjdustableClass`. We could name it `TypeWithCount`.